src/contexts/W3iContext/hooks/w3iProxyHooks.ts

```diff
@@ -22,8 +23,8 @@ export const useW3iProxy = () => {
   )
 
   useEffect(() => {
-    w3iProxy.init()
-  }, [w3iProxy])
+    w3iProxy.init().then(() => setReady(true))
```

Comment:
Could you make `w3iProxy.initialized` a public property and use that instead or do we want a separate piece of state to signal this?

Reply:
Web3inbox initialized is made to true at the start of the init call, to prevent multiple inits (IE from a useEffect rerunning thanks to React strict mode) from recreating clients.

This ready flag is guaranteed to only be true after initialization is fully complete